mock/veda.config.js

```diff
@@ -6,6 +6,10 @@ module.exports = {
 
   pageOverrides: {
     aboutContent: './about.mdx',
-    'sandbox-override': './sandbox-override.mdx'
+    'sandbox-override': './sandbox-override.mdx',
+
+    '/disclaimer': './custom-pages/disclaimer.mdx',
```

Comment:
I imagined in a way that we have one folder (like `custom-pages`), and whichever mdx file is created in this folder auto-magically gets published, so users don't even need to touch veda.config.js. so developers don't need to be pulled in for support. (but honestly, this will happen rarely, so I think the current approach is fine.)

Reply:
+1 to this ☝️

Reply:
I'd also argue that the slash prefix is a hidden behaviour (until you read the docs)